File: CodeSnippets/c codesnippets/21July/prog6.c

#include<stdio.h>
int main() {

	int a = 10;
	if(10L == a)
		printf("10L\n");
	else if(10==a)
		printf("10\n");
	else 
		printf("0");
	return 0;

}
/*
vivek@vivek-HIRAY:~/Desktop/CodeSnippets/c codesnippets/21July$ cc prog6.c 
vivek@vivek-HIRAY:~/Desktop/CodeSnippets/c codesnippets/21July$ ./a.out 
10L
 */

File: CodeSnippets/javaSnippets/5Aug/prog6.java


class Core2web {

	public static void main(String[] args) {

		int var1 = 23;
		double var2 = 23.5;
		boolean b = (var1 = var2);
		System.out.println(b);

	}

}
/*
vivek@vivek-HIRAY:~/Core2Web-TREE-/CodeSnippets/javaSnippets/5Aug$ javac prog6.java 
prog6.java:9: error: incompatible types: possible lossy conversion from double to int
		boolean b = (var1 = var2);
		                    ^
prog6.java:9: error: incompatible types: int cannot be converted to boolean
		boolean b = (var1 = var2);
		                  ^
2 errors
 */
